src/BuildScriptGeneratorCli/Commands/DockerfileCommand.cs

```diff
@@ -90,37 +90,33 @@ internal override int Execute(IServiceProvider serviceProvider, IConsole console
 
             int exitCode;
             var logger = serviceProvider.GetRequiredService<ILogger<DockerfileCommand>>();
-            using (var timedEvent = logger.LogTimedEvent("DockerFileCommand", buildEventProps))
```

Comment:
Why are we not replacing this timed event?

Reply:
Doesn't this mean that we would no longer be logging the dockerfile command to AI, and we would lose all of the buildEventProps data we have created here?

Reply:
We removed this timed event from DockerfileCommand because the dockerfile command is being logged [here](https://github.com/microsoft/Oryx/blob/main/src/BuildScriptGeneratorCli/Commands/CommandBase.cs#L85) anyway.
So, this is actually redundant.

Reply:
aren't we adding a bunch of valuable info with the [buildEventProps](https://github.com/microsoft/Oryx/blob/f9f754e9571c12b724e224ba9f14c6260abd4d8f/src/BuildScriptGeneratorCli/Commands/DockerfileCommand.cs#L82-L89) variable in this method that is currently not being used? 

If we are going to not log that info, then we should delete the buildEventProps object, but i personally feel like it is useful to have the inputs logged?

Reply:
That is a valid point. So, we see more value logging these (would probably show up twice) instead of removing?
@Cormac, since we chatted about this before, do we agree on adding this back instead of removing it?

Reply:
@snehapar9 @pauld-msft we have an existing call in [`DefaultDockerfileGenerator`](https://github.com/microsoft/Oryx/blob/main/src/BuildScriptGenerator/DefaultDockerfileGenerator.cs#L51) that logs a timed event for "GenerateDockerfile", and as a part of it, we include most of the same information in the [`buildEventProps`](https://github.com/microsoft/Oryx/blob/main/src/BuildScriptGenerator/DefaultDockerfileGenerator.cs#L159-L166) that we were previously logging here. We lose out on the arguments provided to the command, but I think we'd generally be in favor of logging what the ultimate combination of build and runtime platforms and tags were generated for the Dockerfile that what was provided by the user. We also pull the platform arguments during this "GenerateDockerfile" timed event, so if we ever felt like we needed to log those, we can easily do so, but I personally don't think we need to at this moment.